src/mpvis/mpdfg/actions.py

```diff
@@ -250,6 +256,7 @@ def save_vis_multi_perspective_dfg(
     format: str = "svg",
     rankdir: str = "TD",
     diagram_tool: str = "graphviz",
+    arc_thickness_by: str = "frequency",
```

Comment:
[REQ]
Use string literals.

```suggestion
    arc_thickness_by: Literal["frequency", "time"] = "frequency",
```